CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - UUIDs are never generated as URNs [#661](https://github.com/stoplightio/prism/pull/661)
 - Core types are now correctly referenced in the HTTP package, restoring the type checks when using the package separately [#701](https://github.com/stoplightio/prism/pull/701)
 - By upgrading Json Schema Faker to the latest version, now the schemas with `additionalProperties:false` / `additionalProperties:true` / `additionalProperties:object` will be correctly handled when dynamic mocking is enabled [#719](https://github.com/stoplightio/prism/pull/719)
+- Making a request to an operation with a `deprecated` parameter is no longer causing Prism to return a 422 response [#731](https://github.com/stoplightio/prism/pull/721)
```

Comment:
```suggestion
- Making a request to an operation with a `deprecated` parameter is no longer causing Prism to return a 422 response [#731](https://github.com/stoplightio/prism/pull/731)
```

Reply:
actually it's [#721], the link itself was good LOL